src/clarity-angular/tabs/vtabs-content.ts

```diff
@@ -0,0 +1,42 @@
+import {Component} from "@angular/core";
+
+
+/**
+ * @class VTabContent
+ *
+ * @description
+ * A child component of VTabs that:
+ * 1. Provides an id property set and used by the VTabs class to map VTabsNavLink items to the corresponding
+ *    VTabsContent items
+ * 2. Provides an activated property that triggers Angular to add/remove the content to the DOM via *ngIf
+ */
+@Component({
+    selector: "clr-vtabs-content",
+    template: `
+        <section class="vtabs-content" *ngIf="activated" role="tabpanel">
```

Comment:
For `First`: I used `aria-hidden: true` b/c it was already built into the css and easy to re-use for the static example. My thinking was not to have a special style for vtabs-content in the static [UI] case. And, I planned on using *ngIf to control which content container was visible for the child component.

For `Second`: `when aria-hidden is used on an element, it removes all relevance to that element.` but you still have to hide it from the user. *ngIf removes the element from the DOM. So, I'm thinking that those are equivalent wrt accessibility. 

My preference would be to keep the *ngIf in the component and remove aria-hidden from the static example since a clarity-ui user will be implementing the hide/show functionality for the active content container.